# Costs from all of the critics for a single trajectory

float64[] costs
float32 v
float32 w
float32 total_cost